src/coreComponents/constitutive/solid/DamageSpectral.hpp

```diff
@@ -116,12 +125,12 @@ class DamageSpectralUpdates : public DamageUpdates< UPDATE_BASE >
 
   inline
   GEOS_HOST_DEVICE
-  virtual real64 getDegradationSecondDerivative( real64 const d ) const override
+  virtual real64 getDegradationSecondDerivative( localIndex const k, real64 const d ) const override
   {
     #if QUADRATIC_DISSIPATION
-    real64 m = m_criticalFractureEnergy/(2*m_lengthScale*m_criticalStrainEnergy);
+    real64 m = m_criticalFractureEnergy[k]/(2*m_lengthScale*m_criticalStrainEnergy);
     #else
-    real64 m = 3*m_criticalFractureEnergy/(8*m_lengthScale*m_criticalStrainEnergy);
+    real64 m = 3*m_criticalFractureEnergy[k]/(8*m_lengthScale*m_criticalStrainEnergy);
     #endif
     real64 p = 1;
     return -2*m*( pow( d, 3 )*(2*m*p*p + m*p + 2*p + 1) + pow( d, 2 )*(-3*m*p*p -3*p) + d*(-3*m*p - 3) + (-m+p+2) )/pow( pow( 1-d, 2 ) + m*d*(1+p*d), 3 );
```

Comment:
use `LvArray::math::pow`

Reply:
@CusiniM We should have a larger discussion about using the math functions in the global namespace that are in the c "math.h" header, which will correlate with the cuda version...but another discussion. For now we should use the LvArray overloads.